ConsoleApp/Scool/Payment.cs

```diff
@@ -0,0 +1,7 @@
+namespace ConsoleApp.Scool;
+
+class Payment
+{
+    private decimal _paymentAmount { get; set; }

```

Comment:
code convention for properties are PascalCase
so it should be
```suggestion
    private decimal PaymentAmount { get; set; }

```